readme.md

```diff
@@ -0,0 +1,2 @@
+# a most excellent reported bot
```

Comment:
I wonder if this should be a subheading instead of a main heading?  Maybe the heading could just be # Reporter Bot?
```suggestion
# Reporter Bot
#### A most excellent bot!
```